joint_limits:
  joint1:
    has_position_limits: true
    min_position: -2.9670
    max_position: 2.9670
    has_velocity_limits: true
    max_velocity: 3.14     # rad/s
    has_acceleration_limits: true
    max_acceleration: 6.28 # rad/s^2

  joint2:
    has_position_limits: true
    min_position: -1.6232
    max_position: 1.6232
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: true
    max_acceleration: 6.28

  joint3:
    has_position_limits: true
    min_position: -2.6704
    max_position: 2.6704
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: true
    max_acceleration: 6.28

  joint4:
    has_position_limits: true
    min_position: -2.7576
    max_position: 2.7576
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: true
    max_acceleration: 6.28

  joint5:
    has_position_limits: true
    min_position: -1.5359
    max_position: 1.5359
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: true
    max_acceleration: 6.28

  joint6:
    has_position_limits: true
    min_position: -2.9670
    max_position: 2.9670
    has_velocity_limits: true
    max_velocity: 3.14
    has_acceleration_limits: true
    max_acceleration: 6.28